src-ts/fs-uv.ts

```diff
@@ -0,0 +1,82 @@
+import { Flags, Fs, fs } from "uv";
+/**
+ * Filesystem Bindings from libuv with minor tweaks.
+ *
+ * Non-blocking calls use callbacks and blocking calls have *Sync postfix.
+ */
+
+/** Node.js style callback with no result value.  This only signals completion or error. */
+export type OnDone = (error?: Error) => void;
+/** Node.js style callback with a single result value.  This signals result or error. */
+export type OnResult<T> = (error?: Error, value?: T) => void;
+/** Node.js style callback with multiple result values.  This signals results or error. */
+export type OnResults<T extends any[]> = (error?: Error, ...values: T) => void;
+
+/**
+ * Used when a libuv callback is required, but the user didn't provide one.
+ * It also helps debugging by causing uncaught exceptions on otherwise silent errors.
+ */
+export const noop: OnDone = (err) => { if (err) { throw err; } };
```

Comment:
should we throw?  Is this the right thing to do?
Also I don't like the name `noop` it's actually a `defaultCallback`

Reply:
The throw is pretty safe, it's run on a new JS stack and so won't hurt any control flow.  It will just trigger an uncaught exception message in the log.